### PR TITLE
chore: bump driver version

### PR DESCRIPTION
**Pull request by karenc-bq**
### Description

Bump driver version to 1.1.0

### CODE REVIEW

**Comment by aaronchung-bitquill** [comment]
Do we also need to update the latest minor version in `Maintenance.md`?
https://github.com/aws/aws-advanced-python-wrapper/blob/main/Maintenance.md

**Comment by aaronchung-bitquill** [comment]
Do we also need to update the driver_info.py as well?
https://github.com/aws/aws-advanced-python-wrapper/blob/main/aws_advanced_python_wrapper/driver_info.py

**Comment by aaronchung-bitquill** [comment]
Not sure if the version in `pyproject.toml` needs to also be updated. 
https://github.com/aws/aws-advanced-python-wrapper/blob/main/pyproject.toml#L3

File: README.md [hunk 1]
```diff
@@ -222,12 +222,14 @@ This `aws-advanced-python-wrapper` is being tested against the following Communi
...
 | Aurora MySQL      | - LTS version, see [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Updates.Versions.html#AuroraMySQL.Updates.LTS) for more details. <br><br> - Latest release, as shown on [this page](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/AuroraMySQL.Updates.30Updates.html). |
 | Aurora PostgreSQL | - LTS version, see [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Updates.LTS.html) for more details. <br><br> - Latest release, as shown on [this page](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraPostgreSQLReleaseNotes/AuroraPostgreSQL.Updates.html).)                        |
 
 The `aws-advanced-python-wrapper` is compatible with MySQL 5.7 and MySQL 8.0 as per MySQL Connector/Python.
+> [!WARNING]\
+> AWS Advanced Python Wrapper version 1.1.0 does not support MySQL version 9.0.0 yet.
```
Comment (hsuamz) [inline README.md:232]:
```suggestion
> Due to recent internal changes with the `v9.0.0` MySQL Connector/Python driver in regards to connection handling, the AWS Advanced Python Wrapper is not recommended for usage with `v9.0.0`. The AWS Advanced Python Wrapper will be updated in the future for `v9.0.0` compatibility with the community driver.
```

**Pull request merged**